sympy/assumptions/assume.py

```diff
@@ -86,80 +131,212 @@ def arg(self):
         x + 1
 
         """
-        return self._args[1]
+        args = self.args
+        if len(args) == 1:
+            return args[0]
+        return args
```

Comment:
This also seems hacky to me. Would it not be better to do:
```
@property
def arg(self):
    # provided for backwards compatibility
    args = self.args
    if len(args) > 1:
        raise Something('deprecated. use predargs. More explanation...')
    return args[0]

@property
def predargs(self):
    return self.args
```
Then we could make sure that only `predargs` is used in library code but maintain backward compatibility for downstream code that possibly uses `arg` but doesn't use polyadic predicates.

Reply:
Ok, I'll try. Personally, I'd prefer `arguments` than `predargs` for generality since this design will be the model for functions as objects.

Reply:
Yes, `arguments` is better.